### PR TITLE
DOC-1623: Fixed minor issues and Markdown -> AsciiDoc conversion issues found by QA

### PR DESCRIPTION
**Pull request by lnewson**
Related Ticket: DOC-1623

Description of Changes:
* Fixed a number of markdown conversion issues.
* Fixed the URLs and default value in the `a11ychecker_issue_url_callback` option.
* Fixed the `insertfile` toolbar appearing twice in the classic demo.
* Fixed a number of places that were missing fullstops.
* Fixed other various minor issues found.

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: modules/ROOT/examples/live-demos/file-picker/index.js [hunk 1]
```diff
@@ -18,19 +18,11 @@ tinymce.init({
     input.setAttribute('type', 'file');
     input.setAttribute('accept', 'image/*');
 
-    /*
```
Comment (lnewson) [inline modules/ROOT/examples/live-demos/file-picker/index.js:21]: I removed this as given we no longer support IE, etc... it seemed unnecessary. As for the other fixes, this is something we missed previously but the `onload` function was using `this` as such this demo no longer works when attempting to pick a local image.

**Review by tiny-ben-tran — APPROVED** [review]

File: modules/ROOT/partials/menu-item-ids/table-menu-items.adoc [hunk 2]
```diff
@@ -1,31 +1,31 @@
-[cols="1,1,2",options="header"]
+##[##cols="1,1,2",options="header"]
```
[the remark this answers is not in the document]
Comment (lnewson) [reply modules/ROOT/partials/menu-item-ids/table-menu-items.adoc:1]: Yeah that shouldn't be like that... I'm not even sure how that happened 🤔

[the remark this answers is not in the document]
Comment (lnewson) [reply modules/ROOT/partials/menu-item-ids/table-menu-items.adoc]: Fixed, thanks for the catch!

File: modules/ROOT/partials/plugins/tinydrive-starter-projects.adoc [hunk 3]
```diff
@@ -13,7 +13,7 @@ The starter projects have their own procedures to get {cloudfilemanager} up and
 | image:netcore.svg[,120,link=https://github.com/tinymce/tinydrive-dotnet-mvc-starter]
 | image:nodejs.svg[,120,link=https://github.com/tinymce/tinydrive-nodejs-starter]
 | image:php.svg[,120,link=https://github.com/tinymce/tinydrive-php-starter]
-| image:java.png[link=https://github.com/tinymce/tinydrive-java-spring-starter]
+| image:java.png[,65,120link=https://github.com/tinymce/tinydrive-java-spring-starter]
```
Comment (ltrouton) [inline modules/ROOT/partials/plugins/tinydrive-starter-projects.adoc:16]: Just noticed this as well
```suggestion
| image:java.png[,65,120,link=https://github.com/tinymce/tinydrive-java-spring-starter]
```

**Review by metricjs — APPROVED** [review]

**Pull request merged**